packages/minter-contracts/ligo/src/fixed_price_sale/fixed_price_sale_market.mligo

```diff
@@ -77,18 +98,20 @@ let buy_token(sale_id, storage: sale_id * storage) : (operation list * storage)
   let amount_ = sale.sale_data.amount in
 
   let tx_nft = transfer_fa2(token_for_sale_address, token_for_sale_token_id, 1n , Tezos.self_address, Tezos.sender) in
-#if !FEE
+  let oplist : operation list = [tx_nft] in 
+#if FEE
+  let fee : nat = percent_of_price_nat (storage.fee.fee_percent, sale_price) in
+  let fee_address : address = storage.fee.fee_address in 
+  let oplist = tx_price_and_fee (money_token_id, money_token_address, fee, fee_address, sale_price, seller, oplist) in 
+#elif PER_SALE_FEE
```

Comment:
It looks like `FEE` and `PER_SALE_FEE` are mutually exclusive macros.
It worth mentioning somewhere in the code commends/documentation

Reply:
Thanks I added documentation